src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -23,6 +24,12 @@ type State = {
   downloadSpeed: number
 }
 
+/**
+ * Displays download information including number of files, size of download, and time to download.
+ * Prefer to use {@link download_list_v2/DownloadDetails} instead, particularly when you have information about file packaging in/eligibility
```

Comment:
I think we're still using the v1 DownloadDetails because the APIs for adding all files in a folder or query doesn't tell us anything about eligibility.

Reply:
Yes